IssueTracker/IssueTracker.FileSystem/Data/IData/IBolbConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace IssueTracker.FileSystem.Data.IData;
+public interface IBolbConfiguration
```

Comment:
rename to IBolbConfigurationFactory

Reply:
<img width="199" alt="image" src="https://user-images.githubusercontent.com/74471664/188216127-17db317b-35e5-4b2b-b85f-22aabc9a30be.png">